From 78bff2d35471b877fe6f13b24d7663d7fe32fc24 Mon Sep 17 00:00:00 2001
From: Juan Gutierrez <juan.gutierrez@nxp.com>
Date: Tue, 6 Sep 2016 16:57:07 -0500
Subject: [PATCH 1243/5242] MLK-13246 ARM: dts: imx: generic support for
 6dqscm 1gb board

commit  80e2fd463029a1fcdd3af2bbb5f665357dcf6e4b from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the generic dtsi configuration support for the SCM i.MX6DQ
QWKS and EVB board with 1GB of DDR memory mapping

Signed-off-by: Juan Gutierrez <juan.gutierrez@nxp.com>
Signed-off-by: Alejandro Sierra <alejandro.sierra@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6dqscm-1gb-fix.dtsi           |   14 ++++++++++++++
 .../boot/dts/imx6dqscm-1gb-interleave-android.dtsi |   14 ++++++++++++++
 2 files changed, 28 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6dqscm-1gb-fix.dtsi
 create mode 100644 arch/arm/boot/dts/imx6dqscm-1gb-interleave-android.dtsi

diff --git a/arch/arm/boot/dts/imx6dqscm-1gb-fix.dtsi b/arch/arm/boot/dts/imx6dqscm-1gb-fix.dtsi
new file mode 100644
index 0000000..606d361
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dqscm-1gb-fix.dtsi
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+/ {
+	memory: memory {
+		linux,usable-memory = <0x10000000 0x20000000
+				       0x80000000 0x20000000>;
+	};
+};
diff --git a/arch/arm/boot/dts/imx6dqscm-1gb-interleave-android.dtsi b/arch/arm/boot/dts/imx6dqscm-1gb-interleave-android.dtsi
new file mode 100644
index 0000000..6edda09
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dqscm-1gb-interleave-android.dtsi
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+/ {
+	memory: memory {
+	    linux,usable-memory = <0x10000000 0x40000000>;
+	};
+
+};
-- 
1.7.9.5

